source/includes/_invoices.md

```diff
@@ -199,7 +199,75 @@ Arguments | Type | Description
 --------- | ---- | ------
 size | `number` | number of items to retrieve
 page | `number` | which page to retrieve. _default page size is 10_
-order_by | `string` | field used for sorting results. If missing default is "-due_date". Could be other parameters in the Invoice model, like "-created" or "status=DONE".
+sort | `string` | field used for sorting results. If missing default is "-due_date". Could be other parameters in the Invoice model, like "-created" or "status=DONE".
```

Comment:
hummm actually we can't search for a specific status but we can sort by `status`